lib/src/Pos/Block/BListener.hs

```diff
@@ -19,19 +19,20 @@ import           Pos.Block.Types       (Blund)
 import           Pos.DB.BatchOp        (SomeBatchOp)
 import           Pos.Ssc.Class.Helpers (SscHelpersClass)
 import           Pos.Util.Chrono       (NE, NewestFirst (..), OldestFirst (..))
+import           Pos.Ssc.GodTossing.Type (SscGodTossing)
 
 class Monad m => MonadBListener m where
     -- Callback will be called after putting blocks into BlocksDB
     -- and before changing of GStateDB.
     -- Callback action will be performed under block lock.
     onApplyBlocks
-        :: forall ssc . SscHelpersClass ssc
-        => OldestFirst NE (Blund ssc) -> m SomeBatchOp
+        :: SscHelpersClass SscGodTossing
```

Comment:
`Pos.Ssc.GodTossing.Type` contains this instance, I think it's better to write `HasConfiguration =>` instead.

Reply:
Good point, thanks.